integration-tests/src/test/java/uk/gov/di/authentication/api/AuthenticationCallbackHandlerIntegrationTest.java

```diff
@@ -275,6 +318,7 @@ void shouldRedirectToIPVWhenIdentityRequired()
 
         assertRedirectToIpv(response, false);
         assertOrchSessionIsUpdatedWithUserInfoClaims();
+        assertInformationStoredForNoSessionService(response);
```

Comment:
This is not new, but I figured I should integration test the existing IPV xbrowser stuff while I'm here